products/mariadb.md

```diff
@@ -14,52 +14,62 @@ command: mysqld --version
 eolColumn: Support Status
 sortReleasesBy: 'releaseCycle'
 releases:
-  - releaseCycle: "10.7"
-    release: 2022-02-14
+-   releaseCycle: "10.7"
+    release: 2021-11-05
```

Comment:
i did only check 10.7.

according to https://mariadb.com/kb/en/release-notes-mariadb-107-series/ the first GA Version was released on 2022-02-09

Reply:
The tags themselves don't seem to have any differentiator for GA/RC releases, and the code just picks up the first tag that matches the release cycle. Updated the mariadb regex to ignore releases in each cycle made before the first GA release (#1207) and this is now resolved.

The date is now `2022-020-08` which is when the release was signed:


```
tag mariadb-10.7.2
Tagger: Daniel Bartholomew <db@dbart.us>
Date:   Tue Feb 8 18:14:32 2022 -0500

MariaDB 10.7.2 release
-----BEGIN PGP SIGNATURE-----

iQIzBAABCgAdFiEEF39AEP5WyjM2MAMF8WVvJMdM0dgFAmIC+WAACgkQ8WVvJMdM
0djJuxAAkUYb3tLJ2lIOjpMkAH/EOk0DVciRAQqGD9dyGfjo1n17COlkClSyvuFz
Gf6NOwq+2813w0ScOAMnBlCK2GGookDY/MBurvMjNJmgSZTlyCwBcL+K2NxvosiO
6bg+Xlcb2ZdbIh/sBEAbF1IqwqkT3Ebq567/mZEm09hqV9mk2Wjb1AEksXZXoqlE
1FC+7yzUVxA7sPzCFHb6ZSENWSPRC+66aVn6Vd8f+yp97ThDNz8fgocLoMyOsVip
+egKgRLi4k87Pu6skrZKPkds1af/pTULwGrsLcA2UUJbTvHeLj/wIaR6PijYEXDP
vUbPv+DZkk6tIe2WDx6OFlq49eldnaETScs13R2IqOl0tkydsh4bF5SWw5Glfuur
aQb6zNg/usihJpfRZC3Bqd4SrAV8V3qXv4YJRDibB7tN1rbM0BLEhKMlOhUoKwQj
JHk9W4u28yTYDq2j/0piZ5cWKh6FZTKpAMJucJkANz00jetXimDbXQsk02jnGCYi
AChuNS6i9pNjcydiUdYQbBqCtonuAmgvkcCTQhGs4tLvIWra2lFeEllOWacME9kK
jMmwDGGa5JkaHiH8iBVASIRsCpioEgIlpYdiR1uRKI54Dc9ZoDT5M1Upfmyog1LW
kLzWsZc2uU/4wmi4pziFVTJRXofTvpDxFrONeEIR8my4HVM3NXw=
=nHJF
-----END PGP SIGNATURE-----
```

https://github.com/MariaDB/server/releases/tag/mariadb-10.7.2, https://api.github.com/repos/MariaDB/server/git/tags/1897d89e7f9be0a3627f83d4fda7008885f2cee6